admin/src/components/Tables/OpenCreationsTable.spec.js

```diff
@@ -122,8 +123,12 @@ describe('OpenCreationsTable', () => {
       })
 
       it.skip('renders the component component-edit-creation-formular', () => {
-        expect(wrapper.find('div.component-edit-creation-formular').exists()).toBeTruthy()
+        expect(wrapper.find('div.component-edit-creation-formular').exists()).toBe(true)
       })
     })
+
+    it('funtion updateUserData', () => {
```

Comment:
```suggestion
    describe('call updateUserData', () => {
```

… then have an `it` with an `expect` that describes the expactation …

Reply:
changed